flink-metrics/flink-metrics-core/src/main/java/org/apache/flink/metrics/groups/SinkWriterMetricGroup.java

```diff
@@ -41,5 +41,5 @@
      * For example, a `volatile` field could be set in the async thread and lazily read in the
      * gauge.
      */
-    <G extends Gauge<Long>> G addCurrentSendTimeGauge(G currentSendTime);
+    void setCurrentSendTimeGauge(Gauge<Long> currentSendTimeGauge);
```

Comment:
I do not know if the signature changes belong to this commit. IMO they should be part of the core commit.